lego/apps/events/models.py

```diff
@@ -776,10 +776,14 @@ def restricted_lookup(self) -> tuple[list[User], list]:
         )
         return [registration.user for registration in registrations], []
 
-    def announcement_lookup(self) -> list[User]:
+    def announcement_lookup(self, exclude_waiting_list: bool) -> list[User]:
         registrations: QuerySet[Registration] = self.registrations.filter(
             status=constants.SUCCESS_REGISTER
         )
+
```

Comment:
pretty crappy pseudocode relating to https://linear.app/abakus-webkom/issue/ABA-856/add-filter-for-specific-groups-registered-for-an-event-in#comment-5fe67f60

```suggestion
if specific_groups:
    registrations = registrations.filter(user => user.groups in specific_groups)
```